wee_slack.py

```diff
@@ -2925,6 +2925,26 @@ def msg_command_cb(data, current_buffer, args):
             channel.send_message(message)
     return w.WEECHAT_RC_OK_EAT
 
+@slack_buffer_required
+def command_channels(data, current_buffer, args):
+    e = EVENTROUTER
+    team = e.weechat_controller.buffers[current_buffer].team
+
+    team.buffer_prnt("    Channels:")
```

Comment:
I meant add spaces before each of the channel/user names, so you can see where the list of channels/users end when more lines are printed. Like this:

```
Channels:
    channel1
    channel2
```

Reply:
sorry, I had misunderstood @trygveaa, i think now it is ok.

Reply:
Now you added padding after the names, not before. That's fine for the users as the presences are now aligned, but not much point for channels where only one value is printed.

But what I meant was adding spaces before the names, so they are indented.